pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -71,18 +90,38 @@ func (q *QueryReconciler) Reconcile(ctx context.Context, req *ctrl.Request) (ctr
 		return res, err
 	}
 
-	return ctrl.Result{}, q.runListRestorePoints(ctx, req)
+	dispatcher := q.makeDispatcher(q.Log, q.Vdb, q.PRunner, q.Password)
```

Comment:
I don't think we need the password. Lets see if we can get by not passing that in. Perhaps make it a pointer so we can easily tell when its not available (and return an error when it should).